manager/integration/tests/test_ha.py

```diff
@@ -2759,8 +2759,9 @@ def test_autosalvage_with_data_locality_enabled(client, core_api, make_deploymen
     create_and_wait_deployment(apps_api, deployment)
 
     pod_names = common.get_deployment_pod_names(core_api, deployment)
-    test_data = generate_random_data(VOLUME_RWTEST_SIZE)
-    write_pod_volume_data(core_api, pod_names[0], test_data, filename='test')
+    data_path = '/data/test'
+    write_pod_volume_random_data(core_api, pod_names[0], data_path, DATA_SIZE_IN_MB_1)
```

Comment:
Will dd really ensure to flush data during writes by default? Or, should we need any extra option to make it work?

Reply:
cc @derekbit

Reply:
Thanks @derekbit for the information:
```
we can add `direct`:
`dd if=/dev/urandom of=1M bs=1M count=1 oflag=direct`
Then, the page cache will be bypassed

and adding `sync` will flush the data to dev before finishing the command:
`dd if=/dev/urandom of=1M bs=1M count=1 oflag=direct,sync`
```

But in our test code we use none of these flags, so actually it's possible that there are some cache issues. I think we can further investigate this and see if any failed test cases could probably suffer from this issue.